herbert_core/classes/MPIFramework/@MessagesFilebased/private/parse_folder_contents_.m

```diff
@@ -95,6 +95,12 @@
     varargout{1} = arrayfun(@get_fext,mess_files,'UniformOutput',false);
 end
 end
+function dn = extract_datenum(f_info)
+dn = f_info.datenum;
```

Comment:
in what situation does the file not have a date?

Reply:
apparently, one of the job on Jenkins failed complaining that datenum is not numeric. I, myself, wander when it could happen.